### DISCUSSION TITLE
Issues with URP tool showing in my URP project as HDRP

### DISCUSSION THREAD

**Discussion by evdrevdr** [soupday/cc_unity_tools_URP#69] [General]
Hey man!
I'm having an issue importing into Unity 2022.3. I might have initally installed the wrong version of the importer, ut after replacing it with the correct URP version, when building materils, my characters just show as pink. Additionally, the importer tool is still labelled as HDRP.

Have you got any tips to ensure my project is using the correct URP package? Are there any cached or temp fles that might be overthrowing the new package?

Thanks heaps,
Damian

**Comment by soupday** [comment]
Did you completely remove the old version?

They are considered different packages and they won't replace or update each other.

**A better solution**: Remove all 1.6.3 versions and install [this version (2.0.0)](https://github.com/soupday/CCiC-Unity-Tools). It's for all pipelines and automatically installs the correct shaders and materials for the pipeline. (At least it should. It could do with more testing)

**Reply by soupday** [reply]
Also make sure you have only one pipeline installed in your project. It will preferentially use HDRP, then URP before falling back to Built-in.
